### ISSUE TITLE
Private repos are unavailable

### ISSUE THREAD

**Issue by destroytoday** [smallwins/bugbot#29]
When listing repos with `/bugbot repos`, private repos are not listed. When trying to add a private repo with `/bugbot repo org/reponame`, Bugbot says `Sorry, Github not showing me :org/:reponame`.

**Comment by brianleroux** [comment]
odd, they are working for me, suspect you may not have enabled 3rd party access for your org:

`https://github.com/organizations/[your org name here]/settings/oauth_application_policy`

?

:sweat:

**Comment by jdelman** [comment]
Having the same issue here. The private repos are on a regular user account, not an organization.

**Comment by destroytoday** [comment]
+1 On a regular user account—not an org.

**Comment by destroytoday** [comment]
FWIW, the settings show that Bugbot has private access: ![](http://dstry.it/e6XD/image.png)

**Comment by brianleroux** [comment]
=(

hm. weird! trying to cook up a way I can get a failing test here to fix :grimacing: 

can you create a private repo and add me to it?

**Comment by jasonvarga** [comment]
I'm showing _some_ private org repos are showing up, some not. I can't work out a pattern.

**Comment by brianleroux** [comment]
Super weird; can you add me to any of the ones not working so I can debug?

**Comment by destroytoday** [comment]
@brianleroux Added.

**Comment by brianleroux** [comment]
Cool, looking into it now

**Comment by brianleroux** [comment]
@destroytoday can you pm me the reponame? nothing showing on github and idk what to look for!

**Comment by brianleroux** [comment]
nevermind! just showed up. back to it :horse_racing:

**Comment by brianleroux** [comment]
Found it. Deployment inbound. Should be fixed in about 5-10mins or so.

**Comment by jdelman** [comment]
Hey @brianleroux - private repos show up in `/bugbot repos` now, but when I try to do `/bugbot repo org/reponame` I get the following error:

`Sorry, Github not showing me .`

**Comment by brianleroux** [comment]
shit. on it: sorry!

**Comment by brianleroux** [comment]
@jdelman just deployed what I think is a fix…can you confirm?

**Comment by jdelman** [comment]
looks good to me.

**Comment by ryanblock** [comment]
💕

**Comment by jasonvarga** [comment]
:+1:

**Comment by gummalam** [comment]
Hey @brianleroux - I just installed bugbot.  And I am trying to use it for an org/repo.  Private repos show up in /bugbot repos - but doesn't work for /bugbot repo org/repo. I get the following error: "Sorry, Github not showing me org/repo".  While googling, I came across this article.  Looking through the issue, it looks like it has been fixed - I still am experiencing it.  Any ideas on why it's not working?
